### ISSUE TITLE
Damp Rag

### ISSUE THREAD

**Issue by Davnik** [HeavensGate/Eternal#374]
Add a damp rag to janitor's closet.
If abused, nerf damp rag to require some water and work for, like, 15 minutes or 10 cleanings. ?-?

**Comment by woothie** [comment]
How can it be abused? What does it even do...?

**Comment by Davnik** [comment]
You spend a few seconds wiping something down, it becomes clean somehow. I feel like it's an infinite use cleaning tool, without the slip of a mop, the use limits (and immediateness) of a soap, and the highly finite uses of a space cleaner.

In all honesty though, it might just be used for running items through a sink, but doing so in a mobile fashion. Either way, if it cleans junk decals (e.g. blood, vomit), it should be limited in uses and require applications of water, for balance and logic. 

Of course, that's debatable, and counter-fun, but janitor was always the tutorial role, and restrictions on janitor-ing have always leaned towards roleplay fun, right?

**Comment by woothie** [comment]
No, I mean. I think it actually doesn't clean anything.

**Comment by Davnik** [comment]
Oh. Well. In that case, it lowers germ level, right?

**Comment by woothie** [comment]
Germ level is essentially nonfunctional or at least improperly implemented. Then again, nothing in this game synergizes. It seems that it cleans blood, the end. It's not even made for the bartender, but for the detective.

**Comment by Davnik** [comment]
Germ level affects medical infections though, right?

**Comment by woothie** [comment]
Welp, nevermind. I assumed it did judging by the clean_blood() proc it runs when attacking something. clean_blood() doesn't just remove blood DNA on the object or tile, it also sets germ level to 0. Disregard what I said, someone actually thought of this.

**Comment by woothie** [comment]
Either way, the janitor should have a rag, even if it cleans blood. Not a huge deal, it's not much effort to go pick up a soap instead. Add a rag to the janitor's closet.

**Comment by Davnik** [comment]
Even if we have to make a list, it should remove some cleanables. Even though vomit is really gross and an only-1-cleaning thing.

**Comment by staticcharged** [comment]
I don't think it will be abused, let's just put a limit on it, such as: it becomes a dirty rag after 3 uses. 
It would absolutely not clean up vomit. It would become a dirty rag immediately and make things worse by spreading drops to the next tile.

The dirty rag can be cleaned by being put through the laundry or with soap and a few seconds at the sink.

**Comment by Davnik** [comment]
New decals: "a little vomit"
explode_vomit()
